saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/util/pb.scala

```diff
@@ -136,9 +140,12 @@ class ProgressBar(_total: Int) extends Output {
     }
     // time left box
     if (showTimeLeft) {
-      val fromStart = (startTime to DateTime.now).millis.toFloat
+      val millis = (startTime to DateTime.now).millis
+      suffix += " elapsed: " + DurationFormatUtils.formatDuration(millis, "HH:mm:ss", true)
+      val fromStart = millis.toFloat
       val left = (fromStart / current) * (total - current)
```

Comment:
@kordjamshidi @umarmanzoor 
Added this to have elapsed time as well. Let me know if you don't want this change.